scripts/requirements.txt

```diff
@@ -1,5 +1,4 @@
 gitpython
 orderedset
-regex
```

Comment:
This entry is duplicated, as it is already in `dateparser/requirements.txt` (and it was causing errors when installing the requirements with tox).